src/assets/styles/abstract/variables.scss

```diff
@@ -2,12 +2,19 @@
 $dark-blue: #090975;
 $light-blue: #3639fc;
 $white: #ffffff;
+$border-gray : #c5c3c36b;
+$background-option-gray : #f7f7f76b;
 
 // screen size
 $break-medium: 576px;
 $break-large: 768px;
 
 // font header
 $font-small: 11px;
-$font-medium: 20px;
+$font-medium: 21px;
 $font-large: 25px;
+
+// title
+$padding-title-large: 50px;
```

Comment:
I don't think variables are needed for this, do you think to use these variables in the future for other pages?